articles/creating-windows-csps.md

```diff
@@ -1,3 +1,5 @@
+# Creating Windows CSPs
+
 ## Introduction
```

Comment:
```suggestion
```

Hey! I think we can cut this top section. The title [here](https://github.com/fleetdm/fleet/pull/26731/files#diff-70f6785686d28046feee3733173883a1b1b22e8c6c32fe5e026eb3cdfec6d098R232) will populate the page when the fleetdm.com website builds:

![Screenshot 2025-03-04 at 9 38 39 AM](https://github.com/user-attachments/assets/8d0df958-8365-460b-8731-7294b21dada8)